src/openrct2/world/Location.hpp

```diff
@@ -45,6 +45,9 @@ struct LocationXYZ16 {
 assert_struct_size(LocationXYZ16, 6);
 #pragma pack(pop)
 
+constexpr sint32 COORDS_NULL = -1;
```

Comment:
Small suggestion here, why not COORDS_INVALID instead of COORDS_NULL as the constant is -1.